SEC/Securities/SecuritiesIdentification.rdf

```diff
@@ -433,7 +398,7 @@
 		<rdfs:subClassOf>
 			<owl:Restriction>
 				<owl:onProperty rdf:resource="&lcc-lr;identifies"/>
-				<owl:someValuesFrom rdf:resource="&fibo-sec-sec-lst;Listing"/>
+				<owl:someValuesFrom rdf:resource="&fibo-sec-sec-lst;ListedSecurity"/>
```

Comment:
No is's more specific than this: it represents the offer from a specific Exchange: the actual Security is referenced only indirectly. ListedSecurity WAS the issued share (in the case of equities) which is available through multiple exchanges, and it may have a different ticker on each. I see you've changed the definition of ListedSecurity to be for a single exchange which could have a broader impact